docker/Dockerfile.indy

```diff
@@ -197,7 +197,7 @@ RUN pip3 install --no-cache-dir \
     -r requirements.bbs.txt \
     -r requirements.dev.txt \
     -r requirements.anoncreds.txt
-RUN curl -sL https://github.com/Indicio-tech/anoncreds-rs/releases/download/v0.1.0-dev.9/library-linux-x86_64.tar.gz | tar -xz -C /usr/local/lib/python3.9/site-packages/anoncreds/
+RUN curl -sL https://github.com/hyperledger/anoncreds-rs/releases/download/v0.1.0/library-linux-x86_64.tar.gz | tar -xz -C /usr/local/lib/python3.9/site-packages/anoncreds/
```

Comment:
Seeing that we now have a build on PyPI that includes bundled binaries (as of June 2; I had missed that until now :slightly_smiling_face:), this separate step for retrieving the binary is no longer necessary and can be removed from each of the Dockerfiles

Reply:
Thanks, that answers a question I was going to pose later. Will remove those commands.